.circleci/config.yml

```diff
@@ -212,6 +212,25 @@ jobs:
       - run:
           command: ./.ci/validation.sh git-diff
 
+  run-inspections:
+    docker:
+      - image: checkstyle/idea-docker:jdk11-idea2022.2.2
+
+    steps:
+      - checkout
+      - run:
+          name: Run inspections
+          command: |
+            mkdir -p .idea
```

Comment:
Remove -p

Reply:
Why? We typically do this in all of our shell scripts, and CI will break if we decide to start checking in idea files in the future.

```
➜  checkstyle git:(issue-12087) ✗ grep -R "mkdir -p" | wc -l                 
25
➜  checkstyle git:(issue-12087) ✗ grep -R "mkdir" | grep -v "mkdir -p"
.github/workflows/pitest.yml:          mkdir staging && cp -r target/pit-reports/ staging
.ci/idea_inspection.bat:mkdir %RESULTS_DIR%
.ci/idea_inspection.bat:mkdir .idea\scopes
.ci/jsoref-spellchecker/whitelist.words:mkdir


```

Reply:
It is required when we need to create whole path(few folders), not just a folder.

Reply:
I did not use `-p` for parent usage, I used it to not fail if folder exists.

Reply:
But know for sure folder does not exist
https://github.com/checkstyle/contribution/commit/b95d0e4ffa008c77cb7a4a82796c2116bed5f1ec

Reply:
removed